lib/autoload.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+class ClassLoader {
+    private $searchPath = [];
+
+    /**
+     * Register path in which to look for a class.
+     */
+    public function registerPath($path) {
+        if (is_string($path)) {
+            $this->searchPath[] = $path;
+        }
+        if (is_array($path)) {
+            array_push($this->searchPath, ...$path);
```

Comment:
I have just double-checked that the splat operator is available from PHP 5.6+, so fine

Reply:
I've checked that before writing it :)